scripts/postinstall.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+stripe postinstall
```

Comment:
will this binary be found if running under just `sh`? I forget if `$PATH` is a local var for shell scripts.

Reply:
this was a non-issue

Reply:
Instead of running a command, could we have this echo something instead?

Reply:
we can, but I made that a command for the following reasons:

BLUF: leveraging the CLI itself to provide a better user experience / less maintenance burden for us

+ the CLI is already written and built to be a cross platform binary, so leveraging the existing features we needed for this feature in the codebase was convenient for us retain this compatibility reliably when postinstalling on each platform
  - I wanted to use colour on the heart in order to make the output more noticeable and pretty for users, while also respecting users' colour settings, which the CLI has functionality to do. An echo alone couldn't do this so we'd be writing custom scripts for each platform to do this.
  - I did want some smart login checking behaviour so that we didn't have to output a string for folks who were just upgrading their version so they don't have more noise to read through once done.
+ the postinstall command allows us to keep the above mentioned  functionality in one place, rather than writing both bash and powershell / cmd scripts and having to maintain these different postinstall behaviours as a result.

Reply:
if we remove the coloured heart feature and smart login checking, we'd be able to get away with the echo, and duplicating the same message in the brew caveat as well.